### PR TITLE
fix While easy adding fields the empty select values throws error

### PR DESCRIPTION
**Pull request by devyagnesh**

### CODE REVIEW

File: apps/web/components/imports/schema/ColumnsTable.tsx [hunk 1]
```diff
@@ -69,40 +80,107 @@ export function ColumnsTable({ templateId }: ColumnsTableProps) {
...
+                                    return newItem;
+                                  }}
+                                  onChange={onChange}
+                                />
+                              )}
+                            />
+                          ) : null}
+                        </>
+                      );
+                    }}
```
Comment (chavda-bhavik) [inline apps/web/components/imports/schema/ColumnsTable.tsx:160]: Please bring this part out of controller.

File: apps/web/components/imports/schema/ColumnsTable.tsx [hunk 2]
```diff
@@ -69,40 +80,107 @@ export function ColumnsTable({ templateId }: ColumnsTableProps) {
...
-                    </ActionIcon>
-                  </Flex>
-                </td>
+
+                  <label htmlFor="isRequired">
+                    <Flex gap={'xs'}>
+                      isRequired
+                      <Checkbox title="Is Required?" {...register('isRequired')} id="isRequired" />
+                    </Flex>
+                  </label>
```
Comment (chavda-bhavik) [inline apps/web/components/imports/schema/ColumnsTable.tsx:168]: If we only use `Checkbox` with title. https://v6.mantine.dev/core/checkbox/#usage

File: apps/web/components/imports/schema/ColumnsTable.tsx [hunk 3]
```diff
@@ -69,40 +80,107 @@ export function ColumnsTable({ templateId }: ColumnsTableProps) {
           },
         ]}
         extraContent={
-          <tr>
+          <tr style={{ borderTop: '0.0625rem solid #373A40' }}>
```
Comment (chavda-bhavik) [inline apps/web/components/imports/schema/ColumnsTable.tsx:83]: increase  rowspan to `5` by providing `<tr rowSpan="5" ... >"

File: apps/web/components/imports/schema/ColumnsTable.tsx [hunk 4]
```diff
@@ -14,20 +14,40 @@ import { EditIcon } from '@assets/icons/Edit.icon';
...
   templateId: string;
 }
 
 export function ColumnsTable({ templateId }: ColumnsTableProps) {
   const [showAddRow, setShowAddRow] = useState(false);
-  const { register, columns, control, handleSubmit, onEditColumnClick, onDeleteColumnClick, isColumnCreateLoading } =
-    useSchema({
-      templateId,
-    });
+  const [isFocuedOnSelect, setIsFocusOnSelect] = useState(false);
```
Comment (chavda-bhavik) [inline apps/web/components/imports/schema/ColumnsTable.tsx:25]: Spelling typo in `isFocuedOnSelect` and `setIsFocusOnSelect`. May be `isSelectFocused` and `setIsSelectFocused` naming will seems good.

File: apps/web/components/imports/schema/ColumnsTable.tsx [hunk 5]
```diff
@@ -14,20 +14,40 @@ import { EditIcon } from '@assets/icons/Edit.icon';
...
+  const typeValue = watch('type');
 
   return (
-    <form onSubmit={handleSubmit}>
+    <form
+      onSubmit={(e) => {
+        e.preventDefault();
+        handleSubmit();
+        setIsFocusOnSelect(false);
+      }}
```
Comment (chavda-bhavik) [inline apps/web/components/imports/schema/ColumnsTable.tsx:49]: Can we try using `useRef` for `isSelectFocused` and bringing `setIsFocusSelect` = `false` in handleSubmit code itselft?

**Review by chavda-bhavik — CHANGES_REQUESTED** [review]

File: apps/web/components/imports/schema/ColumnsTable.tsx (hunk 4 above)
Comment (devyagnesh) [reply apps/web/components/imports/schema/ColumnsTable.tsx]: have corrected the naming conviction

File: apps/web/components/imports/schema/ColumnsTable.tsx (hunk 5 above)
Comment (devyagnesh) [reply apps/web/components/imports/schema/ColumnsTable.tsx]: tried to implement but i throw not assignable error

**Review by chavda-bhavik — APPROVED** [review]

**Pull request merged**